common/document_parser/cli.py

```diff
@@ -79,8 +79,8 @@ def pdf_to_json(
 
 @cli.command(name="pdf-to-json")
 @click.option(
-    '--parser-path',
-    help='A path to an existing parser function',
+    '--doc_extractor-path',
```

Comment:
This should be `--doc-extractor-path` with all dashes.
This change needs to be added to the `pdf_to_json_cmd_wrapper` refactoring the first variable to `doc_extractor_path`. 

If you would like to go a step further you can change the name of the variable in the function for `pdf_to_json` as well, but that is not needed to get the command to work from command line.

Reply:
Hey this has been resolved can you please approve the PR